Python/Setup/swix/Workloads/Microsoft.VisualStudio.Workload.Python.swixproj

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="SetupProjectBefore.settings" />
+
+  <ItemGroup>
+    <Package Include="$(MSBuildProjectName).res.swr" />
+
+    <Dependency Include="Microsoft.PythonTools" />
+    <Dependency Include="Microsoft.PythonTools.Web" />
+    <Dependency Include="Microsoft.PythonTools.Samples">
+      <Type>Optional</Type>
+    </Dependency>
+    <Dependency Include="Microsoft.PythonTools.Uwp">
+      <Type>Optional</Type>
+    </Dependency>
+    <Dependency Include="CPython3x64">
+      <Type>Optional</Type>
+      <Version>[3.5,3.6)</Version>
+    </Dependency>
+    <Dependency Include="CPython2x64">
+      <Type>Optional</Type>
+      <Version>[2.7,2.8)</Version>
+    </Dependency>
+  </ItemGroup>
+
+  <Import Project="SetupProjectAfter.settings" />
+</Project>
```

Comment:
That's just the sideload script, and it's to pick up the new pkgdefs and VSCTs. We're already doing templates properly

---
In reply to: [69644524](https://github.com/Microsoft/PTVS/pull/1396#discussion_r69644524) [](ancestors = 69644524)